e2e/packages/contracts/src/codegen/tables/Multi.sol

```diff
@@ -17,9 +17,13 @@ import { EncodeArray } from "@latticexyz/store/src/tightcoder/EncodeArray.sol";
 import { FieldLayout, FieldLayoutLib } from "@latticexyz/store/src/FieldLayout.sol";
 import { Schema, SchemaLib } from "@latticexyz/store/src/Schema.sol";
 import { PackedCounter, PackedCounterLib } from "@latticexyz/store/src/PackedCounter.sol";
+import { ResourceId } from "@latticexyz/store/src/ResourceId.sol";
+import { RESOURCE_TABLE } from "@latticexyz/store/src/storeResourceTypes.sol";
 
-bytes32 constant _tableId = bytes32(abi.encodePacked(bytes16(""), bytes16("Multi")));
-bytes32 constant MultiTableId = _tableId;
+ResourceId constant _tableId = ResourceId.wrap(
+  bytes32(abi.encodePacked(bytes14(""), bytes16("Multi"), RESOURCE_TABLE))
```

Comment:
could we just hardcode the result (hex) of this like we do for field layout?

Reply:
there wouldn't be a gas advantage (since this is computed statically at compile time) and think it would be more complex / less obvious what's going on if we only render hex here

Reply:
ahh if it's computed at compile time then great!